utils/dashboard.ts

```diff
@@ -26,7 +33,7 @@ export const getDashUrl = ({ collectionSlug, pubSlug, mode, subMode }: GetDashUr
 				.replace(/ /gi, '-')}`
 		: '';
 
-	return `${baseHref}${modeString}${subModeString}${baseQuery}`;
+	return `${baseHref}${modeString}${subModeString}${section && `#${section}`}${baseQuery}`;
```

Comment:
⚠️ The `section` and `baseQuery` need to be swapped here so we don't end up with URLS lik `foo.pubpub.org/dash/blah#blah?blah`